qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/views/CreateCustomerView.groovy

```diff
@@ -63,8 +65,11 @@ class CreateCustomerView extends FormLayout {
         //Generate FormLayout and the individual components
         FormLayout createCustomerForm = new FormLayout()
 
-        this.titleField = new TextField("Title")
+        List<String> academicTitles = AcademicTitle.values().toList().stream().map{ it -> it.getValue() }.collect(Collectors.toList())
```

Comment:
Nice conversion! But Jesus Oo Thanks mister Vaadin

Reply:
Yes we need to clean up the ViewModel usage and put the list of possible values in there somehow